communities.json

```diff
@@ -771,6 +771,14 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "callstack.svg",
+      "title": "Callstack",
```

Comment:
I don't think I can include "Callstack" since it is primarily a company and not an open source project. I could potentially include some of your open source projects if this is the dedicated server for them